pkgs/test/cuda/cuda-samples/generic.nix

```diff
@@ -1,31 +1,46 @@
 { lib
-, cudaPackages
+, backendStdenv
+, cmake
 , fetchFromGitHub
 , fetchpatch
-, addOpenGLRunpath
+, autoAddOpenGLRunpathHook
 , cudatoolkit
+, cudaVersion
 , pkg-config
 , sha256
 , glfw3
 , freeimage
 }:
-cudaPackages.backendStdenv.mkDerivation rec {
+backendStdenv.mkDerivation (finalAttrs: {
   pname = "cuda-samples";
-  version = lib.versions.majorMinor cudatoolkit.version;
+  version = cudaVersion;
 
   src = fetchFromGitHub {
     owner = "NVIDIA";
-    repo = pname;
-    rev = "v${version}";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
     inherit sha256;
   };
 
-  nativeBuildInputs = [ pkg-config addOpenGLRunpath glfw3 freeimage ];
+  nativeBuildInputs = [
+    pkg-config
+    autoAddOpenGLRunpathHook
+    glfw3
+    freeimage
```

Comment:
nit: let's alphabetize these while we're here

Reply:
Resolved in 83d5298d37fd77d812787950f0e16a16b5ecf1bb